src/core/graph.go

```diff
@@ -19,6 +22,9 @@ type BuildGraph struct {
 	packages *cmap.Map[packageKey, *Package]
 	// Registered subrepos, as a map of their name to their root.
 	subrepos *cmap.Map[string, *Subrepo]
+	// Subincludes that are subincluded by other subincludes
+	subincludeSubincludes map[BuildLabel]labelSet
```

Comment:
is there a reason for going with map+mutex rather than cmap as the others? I guess it doesn't need all the notify-on-write stuff, it just feels like an odd one out

Reply:
Because the object in the map here is not atomic. We need to guard against mutating not only the map, but the label set too.

Reply:
Please add a comment to record that